Formula/libngspice.rb

```diff
@@ -0,0 +1,50 @@
+class Libngspice < Formula
+  desc "Spice circuit simulator as shared library"
+  homepage "http://ngspice.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/ngspice/ng-spice-rework/26/ngspice-26.tar.gz"
+  sha256 "51e230c8b720802d93747bc580c0a29d1fb530f3dd06f213b6a700ca9a4d0108"
+
+  head do
+    url "git://ngspice.git.sourceforge.net/gitroot/ngspice/ngspice"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build
+    depends_on "libtool" => :build
+  end
+
+  option "without-cider", "Build without CIDER integration"
+  option "without-xspice", "Build without x-spice extensions"
```

Comment:
I think this depends on the specific use case that people have in mind with using the library. I have no problem with hard-coding them in, but I thought that offering them as options would be most convenient, i.e. _if_ people want to disable them they can do so rather than force them to a specific set of options.